apps/react/src/api/services/animeService.ts

```diff
@@ -0,0 +1,30 @@
+import { AnimeDto } from '@js-camp/core/dtos/anime.dto';
+import { PaginationDto } from '@js-camp/core/dtos/pagination.dto';
+import { AnimeMapper } from '@js-camp/core/mappers/anime.mapper';
+import { PaginationMapper } from '@js-camp/core/mappers/pagination.mapper';
+import { Anime } from '@js-camp/core/models/anime';
+
+import { httpClient } from '..';
+
+/** Anime API. */
+export namespace AnimeService {
+  const ANIME_URL = 'anime/anime/';
+
+  const defaultSearchParams = new URLSearchParams({
+    limit: '10',
+    offset: '0',
+    ordering: 'id',
+  });
+
+  /** Get anime list. */
+  export async function getAnimeList(): Promise<readonly Anime[]> {
```

Comment:
You shouldn't force consumers of your function to work with immutable data. Because if for some reason they would need to mutate array, it wouldn't be possible, but if they can easily make array immutable again.
```suggestion
  export async function getAnimeList(): Promise<Anime[]> {
```